src/objectTranslations/npo02__Households_Settings__c-de.objectTranslation

```diff
@@ -50,5 +50,15 @@
         <label><!-- Use Dated Conversion Rates if available --></label>
         <name>Use_Dated_Conversion_Rates__c</name>
     </fields>
+    <fields>
```

Comment:
The translation files are necessary for new fields, just new objects - unless you're actually providing the final translations for the new fields in this PR.

Reply:
Do you want me to remove these then?  I was just being overzealous, I guess.

Reply:
No need to remove them unless you really want to.

Reply:
It's good to know.  I started doing that as part of my process.  I will discontinue, and only worry about it for objects then.